cobalt/site/docs/reference/starboard/modules/byte_swap.md

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Hmm how come this is back, didn't Garo explicitly delete this one and user.md ?

Reply:
Yeah, this looks weird. `byte_swap.h` is encapsulated in an ifdef. I'll take a look. Perhaps the docsite instructions needs an update.